src/meta/meta_ckpt.c

```diff
@@ -520,32 +516,23 @@ __wt_meta_sysinfo_set(WT_SESSION_IMPL *session)
 	WT_ERR(__wt_timestamp_to_hex_string(session, hex_timestamp,
 	    &S2C(session)->txn_global.meta_ckpt_timestamp));
 #endif
-	WT_ERR(__wt_buf_catfmt(session, buf,
-	    "checkpoint_timestamp=\"%s\"", hex_timestamp));
 
-	update = true;
-	/* Retrieve the metadata for this file. */
-	cfg[2] = NULL;
-	if ((ret =
-	    __wt_metadata_search(session, WT_SYSTEM_URI, &config)) == 0) {
-		cfg[0] = config;
-		cfg[1] = buf->mem;
-	} else if (ret == WT_NOTFOUND) {
-		update = false;
-		cfg[0] = buf->mem;
-		cfg[1] = NULL;
-	} else
-		WT_ERR(ret);
-	/* Replace or insert the system info entry. */
-	WT_ERR(__wt_config_collapse(session, cfg, &newcfg));
```

Comment:
The reason I removed this is that it (a) has no effect today, since there's only ever a single field in the configuration ("checkpoint_timestamp"), and (b) wasn't future proof, because `__wt_config_collapse` expects the first entry in the stack to contain the full set of permitted values.